includes/class-sender.php

```diff
@@ -36,6 +37,41 @@ public static function init() {
 
 		// remote delete posts
 		add_action( 'webmention_delete', array( static::class, 'send_webmentions' ) );
+		\add_action( 'init', array( self::class, 'register_postmeta' ), 11 );
```

Comment:
```suggestion
		self::register_postmeta();
```

I think the `init` function is already triggered on init?